public/team.yaml

```diff
@@ -185,7 +185,17 @@ institutions:
         title: Scientific Curator
         picture: img/team/CarmodyLeigh_headshot_Nov2016.jpg
         alumni: false
+        bio:  Michael is a software engineer, working to make a difference where biology and computer science meet. He specializes in full-stack web applications, scripting, and bioinformatics related technologies such as bioPerl, bioPython, and NGS frameworks. He has additional interests in machine learning and data mining algorithms. He aims to bridge the gap between the vast amounts of data and the need for results or visualizations.
```

Comment:
Oops, an accidentally misplaced insertion here. :bowtie: Leigh has Mike's bio and bio has Leigh's bio. Just swap those two.

Reply:
fixed!

Reply:
Brilliant! Thanks!